src/pages/CropSelector/CropTable/CropTable.js

```diff
@@ -29,6 +29,36 @@ import CropDataRender from './CropDataRender';
 import CropDetailsModal from '../../../components/CropDetailsModal/CropDetailsModal';
 import Legend from '../../../components/Legend/Legend';
 
+const getLegendDataBasedOnCouncil = (councilShorthand = '') => {
```

Comment:
can you move this function into constants.js and import it since it's redefined in the calendar view?